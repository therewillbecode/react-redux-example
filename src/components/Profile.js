import React from "react";
import { Card } from "semantic-ui-react";

const Profile = ({ profile }) => (
  <Card
    centered
    color="grey"
    image={profile.picture}
    header={profile.name}
    meta={profile.nickname}
  />
);

export default Profile;
