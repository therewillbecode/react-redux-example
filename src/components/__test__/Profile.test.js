/* eslint-disable */
import React from "react";
import ReactShallowRenderer from "react-test-renderer/shallow";

import Profile from "../Profile";

const profile = {
  name: "Foo Bar",
  given_name: "Foo",
  family_name: "Bar",
  nickname: "Foobar",
  picture: "/photo.jpg"
};

describe("Profile", () => {
  it("renders correctly", () => {
    const renderer = new ReactShallowRenderer();
    renderer.render(<Profile profile={profile} />);
    const tree = renderer.getRenderOutput();
    expect(tree).toMatchSnapshot();
  });
});
