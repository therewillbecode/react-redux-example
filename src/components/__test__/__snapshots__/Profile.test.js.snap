// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Profile renders correctly 1`] = `
<Card
  centered={true}
  color="grey"
  header="Foo Bar"
  image="/photo.jpg"
  meta="Foobar"
/>
`;
